src/shared/components/query/QueryStore.ts

```diff
@@ -473,20 +417,46 @@ export class QueryStore
 		default: {},
 	});
 
-	@computed get selectableStudiesSet():{[studyId:string]:boolean} {
-		const ret = Object.assign({}, this.cancerStudyIdsSet.result);
-		for (const cohort of this.virtualCohorts) {
-			ret[cohort.id] = true;
-		}
-		return ret;
-	}
+	readonly virtualStudies = remoteData(sessionServiceClient.getUserVirtualStudies(), []);
+
+	readonly selectableIdToPhysicalIdsMap = remoteData<{[studyId:string]:string[]}>({
+		await: ()=>[this.cancerStudies, this.virtualStudies],
+	    invoke: async ()=>{
+			let physicalStudiesIdsMap:{[studyId:string]:string[]} = this.cancerStudies.result.reduce((obj:{[studyId:string]:string[]}, item) =>{
+				obj[item.studyId] = [item.studyId]

```

Comment:
how come the value here is an array of studies?  is this because a virtual study can be composed of multiple physical studies?  but if that's so, i would expect that virtual studies below would allow for multiple elements in the value, but i don't see that.  it just seems like an array of one element.